labeller/backend/utils.js

```diff
@@ -34,11 +34,9 @@ const getAllBucketKeys = async (bucket, prefix) => {
   do {
     const params = {
       Bucket: bucket,
-      Prefix: prefix
+      Prefix: prefix,
+      ...(NextContinuationToken? {ContinuationToken: NextContinuationToken}: {})
```

Comment:
```suggestion
      ContinuationToken: NextContinuationToken
```

Reply:
The way I have it, the property `ContinuationToken` is not added to the object if `NextContinuationToken` is `undefined`.

Your way, the object **will have the property**, but its value will be `undefined`